string tmpl_id
geometry_msgs/Pose pose
---
